internal/moreos/testdata/fake_func-e.go

```diff
@@ -0,0 +1,73 @@
+package main
```

Comment:
nit I would like to name this to `fake_func_e` following the Go convention 😄

Reply:
I have a mixed feeling about this since `func-e` is a name. I will defer this to @codefromthecrypt 🙏🏽.